### PR TITLE
Estate Advertisement role and designation updated

### PR DESCRIPTION
**Pull request by RohanSonthwal**
Estate Advertisement role and designation JSON updated. Please review it once.

### CODE REVIEW

File: data/ga/ACCESSCONTROL-ROLES/rolesdesignations.json [hunk 1]
```diff
@@ -67,6 +67,30 @@
...
+			"rolecode": "ESTD_VERIFIER_GM",
+			"designationCode": "GM",
+			"designationName": "General Manager",
+			"active": true
+		},
+		{
+			"rolecode": "ESTD_APPROVER",
+			"designationCode": "MD",
+			"designationName": "Managing Director",
+			"active": true
```
Comment (SouravDutta-Deloitte) [inline data/ga/ACCESSCONTROL-ROLES/rolesdesignations.json]: Why you need separate Role code for Estate MD?

I think MD is not associated with any Department!!!

If you want to use anywhere role for MD, please use the exsisting role for MD, which you can find above in this file only.
`{
			"code": "GIDC_MD",
			"name": "Goa IDC Managing Director",
			"description": "Goa IDC Managing Director (MD)"
}`

File: data/ga/ACCESSCONTROL-ROLES/roles.json [hunk 2]
```diff
@@ -126,6 +126,26 @@
...
+		},
+		{
+			"code": "ESTD_VERIFIER_GM",
+			"name": "Estate Verifier GM",
+			"description": "Estate Verifier GM"
+		},
+		{
+			"code": "ESTD_APPROVER",
+			"name": "Estate Approver",
+			"description": "Estate Advertisement Approved by MD only"
```
Comment (SouravDutta-Deloitte) [inline data/ga/ACCESSCONTROL-ROLES/roles.json]: as mentioned below, please remove this also.

File: data/ga/ACCESSCONTROL-DEPARTMENTROLES/departmentroles.json [hunk 3]
```diff
@@ -49,6 +49,22 @@
...
+			"departmentcode": "ESTD",
+			"rolecode": "ESTD_VERIFIER_DGM"
+		},
+		{
+			"departmentcode": "ESTD",
+			"rolecode": "ESTD_VERIFIER_GM"
+		},
+		{
+			"departmentcode": "ESTD",
+			"rolecode": "ESTD_APPROVER"
```
Comment (SouravDutta-Deloitte) [inline data/ga/ACCESSCONTROL-DEPARTMENTROLES/departmentroles.json]: same as below

**Review by SouravDutta-Deloitte — CHANGES_REQUESTED** [review]
Please review my comments once.

File: data/ga/ACCESSCONTROL-ROLES/rolesdesignations.json (hunk 1 above)
Comment (RohanSonthwal) [reply data/ga/ACCESSCONTROL-ROLES/rolesdesignations.json]: Okay, we will remove the 'estd_approver'  and use 'gidc_md'  for the same.

File: data/ga/ACCESSCONTROL-DEPARTMENTROLES/departmentroles.json [hunk 4]
```diff
@@ -49,6 +49,18 @@
...
+			"departmentcode": "ESTD",
+			"rolecode": "ESTD_VERIFIER_AM"
+		},
+		{
+			"departmentcode": "ESTD",
+			"rolecode": "ESTD_VERIFIER_DGM"
+		},
+		{
+			"departmentcode": "ESTD",
+			"rolecode": "ESTD_VERIFIER_GM"
```
Comment (SouravDutta-Deloitte) [inline data/ga/ACCESSCONTROL-DEPARTMENTROLES/departmentroles.json]: Also, as discussed in last call,
we do not want to mention any designation details in the roles.

So, please remove the AM, DGM and GM keywords from the Roles.
And as per your level, you can add L1, L2, L3.

For example: 
{
		"departmentcode": "ESTD",
		"rolecode": "ESTD_VERIFIER_L1"
}

File: data/ga/ACCESSCONTROL-ROLES/roles.json [hunk 5]
```diff
@@ -126,6 +126,22 @@
...
+		},
+		{
+			"code": "ESTD_VERIFIER_DGM",
+			"name": "Estate Verifier DGM",
+			"description": "Estate Verifier DGM"
+		},
+		{
+			"code": "ESTD_VERIFIER_GM",
+			"name": "Estate Verifier GM",
+			"description": "Estate Verifier GM"
```
Comment (SouravDutta-Deloitte) [inline data/ga/ACCESSCONTROL-ROLES/roles.json]: please update as above mentioned.

File: data/ga/ACCESSCONTROL-ROLES/rolesdesignations.json [hunk 6]
```diff
@@ -67,6 +67,24 @@
...
+			"rolecode": "ESTD_VERIFIER_DGM",
+			"designationCode": "DGM",
+			"designationName": "Deputy General Manager",
+			"active": true
+		},
+		{
+			"rolecode": "ESTD_VERIFIER_GM",
+			"designationCode": "GM",
+			"designationName": "General Manager",
+			"active": true
```
Comment (SouravDutta-Deloitte) [inline data/ga/ACCESSCONTROL-ROLES/rolesdesignations.json]: please update as mentioned above

**Review by SouravDutta-Deloitte — CHANGES_REQUESTED** [review]
Please refactor.

File: data/ga/ACCESSCONTROL-ROLES/rolesdesignations.json (hunk 6 above)
Comment (RohanSonthwal) [reply data/ga/ACCESSCONTROL-ROLES/rolesdesignations.json]: Updated. Please review.

**Review by SouravDutta-Deloitte — APPROVED** [review]
Thanks.

**Pull request merged**